spec/services/movement_service_spec.rb

```diff
@@ -189,4 +189,135 @@
       expect(processed).to be false
     end
   end
+
+  describe "processing offenders moved to/from immigration estates" do
+    before do
+      create(:allocation, nomis_offender_id: 'G4273GI')
+    end
+
+    let(:allocation) { Allocation.find_by(nomis_offender_id: 'G4273GI') }
+
+    let(:immigration_movement) do
+      create(:movement, offender_no: 'G4273GI', direction_code: direction_code, create_date_time: Date.new(2020, 1, 6),
+                        movement_type: movement_type, from_agency: from_agency, to_agency: to_agency)
+    end
+
+    context 'when movement is a transfer' do
+      let(:case_info) { create(:case_information, nomis_offender_id: 'G4273GI') }
+
+      context 'when the from_agency is MHI' do
```

Comment:
I'd write out what MHI means in full here, which I think is 
```ruby
context 'when the movement is to an immigration centre' do
``` 
or similar?

Reply:
I don't think this is necessary as it is explained in the code 🤷‍♀ but again, happy to discuss